### DISCUSSION TITLE
document uuid

### DISCUSSION THREAD

**Discussion by cellmechanic** [outline/outline#7705] [Self Hosting]
Hey, I'm writing a simple discord notification thing using the webhooks, and I'm trying to get revision history so I can show part of updates and stuff using a diff model. I can't for the life of me figure out what it means when it says the documents UUID, I've tried every ID  that comes in the JSON and they all get errors about invalid UUID, what am I doing wrong??

**Comment by tommoor** [comment]
>  what am I doing wrong??

Provide some code and then it'll be easier to figure out
